sdc/datatypes/hpat_pandas_dataframe_rolling_functions.py

```diff
@@ -307,6 +307,35 @@ def sdc_pandas_dataframe_rolling_count(self):
     return gen_df_rolling_method_impl('count', self)
 
 
+@sdc_overload_method(DataFrameRollingType, 'cov')
+def sdc_pandas_dataframe_rolling_cov(self, other=None, pairwise=None, ddof=1):
+
+    ty_checker = TypeChecker('Method rolling.cov().')
+    ty_checker.check(self, DataFrameRollingType)
+
+    accepted_other = (Omitted, NoneType, DataFrameType, SeriesType)
+    if not isinstance(other, accepted_other) and other is not None:
+        ty_checker.raise_exc(other, 'DataFrame, Series', 'other')
+
+    accepted_pairwise = (bool, Boolean, Omitted, NoneType)
+    if not isinstance(pairwise, accepted_pairwise) and pairwise is not None:
+        ty_checker.raise_exc(pairwise, 'bool', 'pairwise')
+
+    if not isinstance(ddof, (int, Integer, Omitted)):
+        ty_checker.raise_exc(ddof, 'int', 'ddof')
+
+    none_other = isinstance(other, (Omitted, NoneType)) or other is None
+    kws = {'other': 'None', 'pairwise': 'None', 'ddof': '1'}
+
+    if none_other:
+        return gen_df_rolling_method_other_none_impl('_df_cov', self, kws=kws)
```

Comment:
I think we should add a comment here to explain the need for _df_cov (or maybe add docstrings to cov and _df_cov overloads).

Reply:
Let me add a comment here.